### PR TITLE
tweak some error messages to be more like WMA...

### PR DESCRIPTION
**Pull request by rocky**
in particular Get::noopen, {First,Last,Most}::normal

### CODE REVIEW

File: mathics/builtin/list/eol.py [hunk 1]
```diff
@@ -589,24 +590,24 @@ class First(Builtin):
     >> First[a + b + c]
      = a
     >> First[x]
-     : Nonatomic expression expected.
+     : Nonatomic expression expected at position 1 in First[x].
      = First[x]
     >> First[{}]
      : {} has zero length and no first element.
      = First[{}]
     """
```
Comment (axkr) [inline mathics/builtin/list/eol.py:599]: `First` and `Last`should have a default value as second arg and attribute `A_HOLD_REST`.

Comment (mmatera) [reply mathics/builtin/list/eol.py:599]:
```
In[1]:= First[a]                                                                

First::normal: Nonatomic expression expected at position 1 in First[a].

Out[1]= First[a]

In[2]:= First[a, b]                                                             

Out[2]= b

In[3]:= First[a, b, c]                                                          

First::argt: First called with 3 arguments; 1 or 2 arguments are expected.

Out[3]= First[a, b, c]

```

Comment (rocky) [reply mathics/builtin/list/eol.py:611]: Thanks -  this should be handled now.

File: mathics/builtin/list/eol.py [hunk 2]
```diff
@@ -580,39 +581,74 @@ class First(Builtin):
...
+            return
+
+        first_elem = expr.elements[0]
+
+        if expr.head == SymbolSequence or (
+            not isinstance(expr, ListExpression)
+            and len == 2
+            and isinstance(first_elem, Atom)
+        ):
+            return expr.elements[1].evaluate(evaluation)
```
Comment (mmatera) [inline mathics/builtin/list/eol.py:649]: Just a comment: to call the `evaluate(evaluation)` method at the end is not needed: it is going to be called in the next `rewrite_apply_eval` iteration

Comment (rocky) [reply mathics/builtin/list/eol.py:649]: Ok - thanks. Will remove the explicit call.

**Pull request merged**